rholang/src/main/scala/coop/rchain/rholang/interpreter/compiler/normalize.scala

```diff
@@ -66,38 +65,40 @@ object GroundNormalizeMatcher {
 }
 
 object RemainderNormalizeMatcher {
-  def handleProcVar[M[_]](pv: ProcVar, knownFree: DebruijnLevelMap[VarSort])(
+  def handleProcVar[M[_]](pv: ProcVar, knownFree: DeBruijnLevelMap[VarSort])(
```

Comment:
On multiple methods `M[_]: Sync` can be simplified without "normal" implicit, if you are for more cleanup.

Reply:
That'll be in the next PR.